zc_install/sql/install/mysql_zencart.sql

```diff
@@ -885,6 +885,7 @@ CREATE TABLE manufacturers (
   manufacturers_image varchar(255) default NULL,
   date_added datetime default NULL,
   last_modified datetime default NULL,
+  featured tinyint,
```

Comment:
Shouldn't this have a default value?

Reply:
Yes, while core code won't have any issue with it setting a default will allow custom code that doesn't account for the column to not throw errors.
Thanks.